frontend/controller/actions/group.js

```diff
@@ -213,6 +220,18 @@ export default (sbp('sbp/selectors/register', {
             },
             data: inviteKeyP
           }
+          /* ,
+           // TODO: If this is needed, explain why this is wrapped
+           encryptedOutgoingDataWithRawKey(CEK, {
+            foreignKey: `sp:${encodeURIComponent(userID)}?keyName=${encodeURIComponent('csk')}`,
+            id: userCSKid,
+            data: rootState[userID]._vm.authorizedKeys[userCSKid].data,
+            permissions: [GIMessage.OP_ACTION_ENCRYPTED + '#inner'],
+            allowedActions: '*',
+            purpose: ['sig'],
+            ringLevel: Number.MAX_SAFE_INTEGER,
+            name: `${userID}/${userCSKid}`
+          }) */
```

Comment:
If this is no longer needed can it be removed?

Reply:
It can